test/format/format_config.c

```diff
@@ -496,6 +496,19 @@ config_run(void)
           REALLOC_MAX_TABLES);
     }
 
+    if (GV(RUNS_PREDICTABLE_REPLAY)) {
+        /*
+         * Predictable replays can get extremely slow with throttling.
+         *
+         * TODO: Investigate reasons for that.
```

Comment:
Can we have a fixme comment with a ticket id instead here of the todo here please?

Reply:
If there's already a new ticket, I agree.  If not, I'm okay with dropping the comment and not having a ticket.  I think we know already: that the interaction is slow, that we don't understand completely why, but it's likely to be a little complex, and that it doesn't matter that much.  Without throttling, we'd do a slightly less rigorous testing of predictable replay, but PR isn't a typical way of running WT.

Reply:
Added ticket number: WT-11782